code/modules/mob/mob_movement.dm

```diff
@@ -75,7 +75,11 @@
 
 	if(!mob.Process_Spacemove(direct))
 		return FALSE
-
+	if (!src.prefs.bindings.isheld_key("Ctrl"))
```

Comment:
if (!src.prefs.bindings.isheld_key("Ctrl") && !(mob.datum_flags & DF_VAR_EDITED))
i want pixel shifted Ian thank you very much

Reply:
thats shit code gamer

Reply:
nah